penguin/web/services/provider_catalog.py

```
@@ -219,6 +228,134 @@ def models_dev_provider_models(
+
+    discovered: dict[str, dict[str, Any]] = {}
+    try:
+        with httpx.Client(timeout=10.0) as client:
+            response = client.get(
+                _OPENAI_CODEX_MODELS_URL,
+                headers=headers,
+                params={"client_version": _OPENAI_CODEX_MODELS_CLIENT_VERSION},
+            )
+            response.raise_for_status()
+            payload = response.json()
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)